man/run0.xml

```diff
@@ -219,6 +219,35 @@
     non-zero return value will be returned.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Environment Variables</title>
+
+    <para>As with <literal>systemd-run</literal>, the session will inherit the system
+    environment from systemd. In addition, the following environment variables will be set:</para>
```

Comment:
from systemd → from the service manager.

i.e. "systemd" is both a service manager and a suite of components. i think it makes sense to be clear here and say service manager.